source/js/classes/crop-host.js

```diff
@@ -379,22 +398,20 @@ crop.factory('cropHost', ['$document', 'cropAreaCircle', 'cropAreaSquare', 'crop
         theArea = new CropAreaCircle(ctx, events);
 
         // Init Mouse Event Listeners
-        $document.on('mousemove', onMouseMove);
+        elCanvas.on('mousemove', onMousehover);
```

Comment:
onMousehover is necessary for changing some of the overlay styling as your mouse moves over the canvas. E.g. the cursor changing style.

onMouseMove is something separate that does the heavy work of drawing the scene (drawScene()) only when the user has moused down and also moved the cursor over the canvas. This is to prevent ngcrop code being run when, for example, a user clicks on inputs in the Vivopad.

As far as I can tell the resizing functionality behaves as it should.